### PR TITLE
feat(system): Add Jupyter

### PR DESCRIPTION
**Pull request by sfoster1**
Add Jupyter, its configuration, and a whole bunch of its dependencies. In addition, configure it and start it on boot.

There are a whole lot of dependencies here added by utils/scanpypi (there are also some modifications to utils/scanpypi that make it a little more able to handle projects' weird setuptools invocations). There are a whole lot of modifications to these dependencies to make sure that they actually get their transitive dependencies in, since some of the jupyter projects have the bad habit of including themselves from their setup file.

Finally, there's the unit file that runs jupyter. This is a bit more in depth than the old system, in part because it's an actual unit file and in part because jupyter is actually installed under its own service user (read: no home directory, no default shell) so you need to tweak some config defaults that won't work.

Blocked by #9 

Relates to https://github.com/Opentrons/opentrons/issues/2873

### CODE REVIEW

**Review by btmorr — APPROVED** [review]
Looks good, functional tests to come once the API is installed

File: package/Config.in [hunk 1]
```diff
@@ -809,6 +809,7 @@ menu "External python modules"
 	source "package/python-bcrypt/Config.in"
 	source "package/python-beautifulsoup4/Config.in"
 	source "package/python-bitstring/Config.in"
+  source "package/python-bleach/Config.in"
```
Comment (mcous) [inline package/Config.in]: Nitpick, but looks like there's some whitespace issues in this file?

File: utils/scanpypi [hunk 2]
```diff
@@ -272,31 +278,69 @@ class BuildrootPackage():
...
+        return self.setup_args[self.metadata_name]
+
     def load_setup(self):
         """
         Loads the corresponding setup and store its metadata
         """
         current_dir = os.getcwd()
         os.chdir(self.tmp_extract)
         sys.path.append(self.tmp_extract)
+        print("FINDING SETUP WITH PATH {}".format(sys.path))
```
Comment (mcous) [inline utils/scanpypi]: Looks like a few `print`s were added. Just want to double-check that they were intentionally left in

**Review by mcous — APPROVED** [review]
🐺 LGTM

**Pull request merged**